build.gradle.kts

```diff
@@ -61,7 +61,9 @@ tasks.register("unitTestAll") {
         ":tools:unit:testReleaseUnitTest",
         ":sample:java:assembleDebug",
         ":sample:kotlin:assembleDebug",
-        ":sample:kotlin-timber:assembleDebug"
+        ":sample:kotlin-timber:assembleDebug",
```

Comment:
Just nitpicking here, but it'd be better to keep those sorted, so moving `dd-sdk-android-ktx` just before `dd-sdk-android-timber`

Reply:
only reason I didn't do this was to not create conflicts with my other PR, lazy :) but yeah I will keep them sorted